### PR TITLE
Search and feedback bar removed

### PR DESCRIPTION
**Pull request by apmcguigan**
https://trello.com/c/FC4LD94P

https://trello.com/c/xmcZ8NbW

### CODE REVIEW

**Comment by GBGRAVES** [comment]
[Disable / remove search bar](https://trello.com/c/FC4LD94P/269-disable-remove-search-bar)

File: components/paper.js [hunk 1]
```diff
@@ -4,6 +4,7 @@ import PropTypes from "prop-types";
 import { css, jsx } from "@emotion/core";
 import { globalTheme } from "../theme";
 import VersionBanner from "./version_banner";
```
Comment (thomascraig) [inline components/paper.js]: Do you still require VersionBanner ?

File: components/paper.js [hunk 2]
```diff
@@ -28,7 +29,7 @@ class Paper extends Component {
       <div>
         <div css={this.bannerStyle}>
```
Comment (thomascraig) [inline components/paper.js:32]: Consider add green css class to parent div?

File: components/paper.js [hunk 3]
```diff
@@ -4,6 +4,7 @@ import PropTypes from "prop-types";
 import { css, jsx } from "@emotion/core";
 import { globalTheme } from "../theme";
 import VersionBanner from "./version_banner";
+import GreenBanner from "./green_banner";
```
Comment (thomascraig) [inline components/paper.js]: Should be able to remove, if add class to parent div.

File: __tests__/components/benefit_pane_test.js [hunk 4]
```diff
@@ -224,21 +224,11 @@ describe("BenefitsPane", () => {
       });
     });
 
-    describe("search feature", () => {
-      it("shows a text search box", () => {
```
Comment (thomascraig) [inline __tests__/components/benefit_pane_test.js:228]: Particular reason for removing the validation of the search box?

**Review by thomascraig — COMMENTED** [review]
Couple suggestions.

File: __tests__/components/benefit_pane_test.js (hunk 4 above)
Comment (apmcguigan) [reply __tests__/components/benefit_pane_test.js:228]: Removed this test since it was trying to find the search bar which has been removed from the page.

**Review by thomascraig — APPROVED** [review]
Nice work.

**Pull request merged**